lib/stm32/f4/rcc.c

```diff
@@ -709,14 +693,14 @@ uint32_t rcc_system_clock_source(void)
 
 void rcc_clock_setup_hse_3v3(const struct rcc_clock_scale *clock)
 {
-	/* Enable internal high-speed oscillator. */
+	/* Enable internal high-speed oscillator (HSI). */
```

Comment:
Okay, I was just trying to improve the comments. Should I edit the commit to remove those, and submit a separate PR? Or just nix them entirely?